packages/angular_devkit/build_ng_packagr/README.md

```diff
@@ -1,3 +1,3 @@
 # Angular Build Architect for ng-packagr
 
-WIP
\ No newline at end of file
+This package has been deprecated. Please use `NgPackagrBuilderOptions` from `@angular-devkit/build-angular` instead.
```

Comment:
```suggestion
This package has been deprecated. Please use the `ng-packagr` builder and `NgPackagrBuilderOptions` from `@angular-devkit/build-angular` instead.
```

Reply:
I actually suggested to just to mention the builder, as otherwise we’ll need to
mention all public api methods and interfaces

Reply:
sounds good.

Reply:
Thanks for the edit! Sorry I did this in a hurry and didn't read carefully.